Controllers/AbstractControllers/AbstractIndexController.php

```diff
@@ -33,5 +33,14 @@ public function indexAction(): void
         $this->defaultView->render($options);
     }
 
+    protected function exceptionCatcher(
```

Comment:
There are two questions: 
1. Do we need this function to be created?
2. Function name is explicit